resource/perl_script/cStringifyFile.pl

```diff
@@ -0,0 +1,169 @@
+#########################################################################
```

Comment:
- Modify perl_script to scripts
- File a github issue to move java_scripts to scripts/java_execution_scripts

Reply:
Done